rust/src/applayertemplate/template.rs

```diff
@@ -578,20 +528,55 @@ mod test {
         let buf = b"5:Hello3:bye";
 
         let r = state.parse_request(&buf[0..0]);
-        assert_eq!(r, AppLayerResult{ status: 0, consumed: 0, needed: 0});
+        assert_eq!(
```

Comment:
can this be avoided? Its so unnecessarily verbose

Reply:
This appears to be dependent on the size of the struct literal.  With Rust stable we can use `struct_lit_width = 36` (the default is 18 to get these particular lines to format like:
```
assert_eq!(r, AppLayerResult { status: 0, consumed: 0, needed: 0 });
```
bigger ones will continue to wrap like the above.

This works in current Rust stable, but not as far back as 1.41.1.

Reply:
This option requires Rust 1.53.0 or newer (2021-06-17).  But we can instruct rustfmt to skip assert_eq macros.